ferrocene/doc/target-names.toml

```diff
@@ -22,7 +22,6 @@ thumbv7em-none-eabihf       = "Armv7E-M bare-metal (hard-float)"
 "thumbv8m.base-none-eabi"   = "Armv8-M Baseline bare-metal"
 "thumbv8m.main-none-eabi"   = "Armv8-M Mainline (soft-float)"
 "thumbv8m.main-none-eabihf" = "Armv8-M Mainline (hard-float)"
-wasm32-unknown-unknown      = "WASM bare-metal"
```

Comment:
I would keep the entry in `target-names.toml`, in case we still need to reference it (like in the relnotes). I'd add a `# Removed targets:` section at the end and move the target there.

Reply:
I'll reflect it in #1267 as well